asv/benchmarks/setup/bench_model.py

```
@@ -81,6 +91,11 @@ def peakmem_initialize_model_cpu(self, robot, num_envs):
 
         del model
 
+    @skip_benchmark_if(wp.get_cuda_device_count() == 0)
+    def time_initialize_solver(self, robot, num_envs):
```

Ruff (0.12.2)

Unused method argument: `num_envs`

(ARG002)